docs/conf.py

```diff
@@ -138,11 +137,12 @@
 # Add any paths that contain custom themes here, relative to this directory.
 # To use a different custom theme, add the directory containing the theme.
 #html_theme_path = []
-
 # The theme to use for HTML and HTML Help pages.  See the documentation for
```

Comment:
Can you get rid of any remaining non-required changes? (i.e. add back the empty lines that were removed)

Reply:
Yup, done